native/src/routes/Pois.tsx

```diff
@@ -88,7 +82,7 @@ const Pois = ({ pois, language, cityModel, route, navigation }: PoisProps): Reac
     }, RESTORE_TIMEOUT)
   }
   const selectPoiFeature = (feature: PoiFeature | null) => {
-    if (feature && cameraRef.current) {
+    if (feature) {
```

Comment:
@f1sh1918 just to make sure: This was unnecessary, right?

Reply:
looks okay to me. couldn't find any issue while testing